### PR TITLE
Fixed #1854,#1858,#1859,#1860 Styling details

### PR DESCRIPTION
**Pull request by martinb14**
http://platform.dandigbib.org/issues/1854
http://platform.dandigbib.org/issues/1858
http://platform.dandigbib.org/issues/1859
http://platform.dandigbib.org/issues/1860

### CODE REVIEW

**Comment by cableman** [comment]
The commit message should follow:

> [issue number]: [short summary of change]. The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

If a single commit fixed more issue the main issue is ref. and the issues are link in the issue tracker.

**Comment by cableman** [comment]
The commit contains a lot more that just styling changes.

Please split this up-into commit that match a given message.

File: modules/p2/ding_list/ding_list.module [hunk 1]
```diff
@@ -594,12 +594,12 @@ function ding_list_preprocess_entity(&$variables) {
         }
         $objects = ding_base_get_items('ding_type', $variables['ding_type'], 'field_list_objects');
         $count = count($objects);
-        $variables['title'] = format_plural(
-          $count,
-          '@list_title (1)',
-          '@list_title (@count)',
-          array('@list_title' => $variables['title'])
-        );
+        // $variables['title'] = format_plural(
```
Comment (cableman) [inline modules/p2/ding_list/ding_list.module]: Please remove comment-out code.

**Comment by DDBdrift** [comment]
https://platform.dandigbib.org/issues/1854 
https://platform.dandigbib.org/issues/1858
https://platform.dandigbib.org/issues/1859 
https://platform.dandigbib.org/issues/1860 | Resolved (tag version)

**Pull request closed without merging**